### PR TITLE
Further code cleanup

### PR DESCRIPTION
**Pull request by chenglou**
It's unclear whether this is more readable, but whatever

### CODE REVIEW

**Comment by chenglou** [comment]
Before:
Screen Shot 2021-02-25 at 9 54 29 PM

After:

File: src/res_scanner.ml [hunk 1]
```diff
@@ -133,23 +133,27 @@ let make ~filename src =
...
   let startOff = scanner.offset in
-  while (
-    CharacterCodes.isLetter scanner.ch ||
-    CharacterCodes.isDigit scanner.ch ||
-    '_' == scanner.ch ||
-    '\'' == scanner.ch
-  ) do
-    next scanner
-  done;
+  let rec skipGoodChars scanner =
```
Comment (IwanKaramazow) [inline src/res_scanner.ml:149]: better assembly?

**Review by IwanKaramazow — APPROVED** [review]

File: src/res_scanner.ml (hunk 1 above)
Comment (IwanKaramazow) [reply src/res_scanner.ml:149]: Vs while loop

Comment (chenglou) [reply src/res_scanner.ml:149]: No and I'm not optimizing at this level anymore, but you'll see

**Pull request merged**